test/extended/builds/run_fs_verification.go

```diff
@@ -0,0 +1,136 @@
+package builds
+
+import (
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-builds][Feature:Builds] verify /run filesystem contents", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc                                      = exutil.NewCLI("verify-run-fs")
+		testVerityRunFSWriteableBuildConfigYaml = fmt.Sprintf(`
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: verify-run-fs
+spec:
+  runPolicy: Serial
+  source:
+    dockerfile: "FROM %s\nRUN chmod -R uga+rwx /run\nUSER 1001"
+    type: Dockerfile
+  strategy:
+    dockerStrategy:
+      env:
+      - name: BUILD_LOGLEVEL
+        value: "10"
+      imageOptimizationPolicy: SkipLayers
+    type: Docker
+`, "image-registry.openshift-image-registry.svc:5000/openshift/tools:latest")  // replace with image.ShellImage()) when github.com/openshift/origin/test/extended/util/image lands in this release
+		testVerifyRunFSContentsBuildConfigYaml = fmt.Sprintf(`
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: verify-run-fs
+spec:
+  runPolicy: Serial
+  source:
+    dockerfile: "FROM %s\nRUN ls -R /run\nUSER 1001"
```

Comment:
nit - use multiline YAML
```suggestion
    dockerfile: |
      FROM %s
      RUN ls -R /run
      USER 1001"
```